jerry-core/jerry-port.h

```diff
@@ -35,6 +41,18 @@ int jerry_port_logmsg (FILE *stream, const char *format, ...);
 int jerry_port_errormsg (const char *format, ...);
 int jerry_port_putchar (int c);
 
+void jerry_port_abort (void);
+
+
+/**
+ * Target port functions for memory management
+ */
+mem_heap_t *jerry_port_init_heap (void);
```

Comment:
I think we should add a size argument to this function. At least the implementation can check the size.